src/features/game/expansion/components/PirateQuest.tsx

```diff
@@ -76,14 +76,9 @@ export const PirateQuest: React.FC = () => {
             }}
           />
         )}
-        <NPC
-          onClick={openQuest}
-          pants="Pirate Pants"
-          body="Pirate Potion"
-          hair="Sun Spots"
-          shirt="Striped Blue Shirt"
-          hat="Pirate Hat"
-        />
+        <div className="-scale-x-100">
```

Comment:
I've flipped the quest NPC so they are facing the island instead of looking out to the distance.